packages/MorphicAPIExplorer-Core.package/MEXToolExplorationView.class/instance/changeCurrentMorph.withSelectedMethods..st

```diff
@@ -0,0 +1,8 @@
+default morphs
+changeCurrentMorph: aMorph withSelectedMethods: aMethodList
```

Comment:
We changed this in one of our branches to "changeCurrentMorph: aMorph" only because we don't think there is an actual use case for it, see [here](https://github.com/hpi-swa-teaching/MorphicAPIExplorer/blob/7dfd0e368d4f17d5bab5ea9cd3362f1ffca46371/packages/MorphicAPIExplorer-Core.package/MEXToolExplorationView.class/instance/changeCurrentDefaultMorph..st). Hope it doesn't break anything!